README.md

```diff
@@ -18,3 +18,21 @@ The Windows Azure transports for NServiceBus enable the use of Windows Azure Que
 - [Azure Storage Persistence](https://github.com/Particular/NServiceBus.Persistence.AzureStorage)
 - [Azure Cloud Services Host](http://docs.particular.net/nservicebus/hosting/cloudservices-host/)
 - [Samples](http://docs.particular.net/samples/azure/)
+
+## Third Party Dependency Update Policy
+
+The azure related nuget packages depend on third party packages. As each of these packages is maintained by different parties in different ways, our update policy regarding them also varies.
+
+The following aspects are taken into account:
+
+- Third party follows SemVer
+- API surface size
+- Past experience with behavioral changes in the dependency
+
+
+| Dependency                   | Current Policy                   |  Suggested Future Policy          | 
```

Comment:
Let's add a column that states what versions are those.
We could borrow parts of [the script](https://github.com/Particular/docs.particular.net/blob/master/tools/supportedVersions.linq) that generates the [supported versions](https://docs.particular.net/nservicebus/upgrades/supported-versions) doco.